include/logger/loggerApi.h

```diff
@@ -25,8 +25,12 @@
 
 #define LOGGER_API \
```

Comment:
A separate but related thought comes to mind... if this define is in the header and the header is included in more than one place, does the code size grow by the number of times this is defined?